components/style/properties/shorthand/box.mako.rs

```diff
@@ -363,12 +363,11 @@ macro_rules! try_parse_one {
                     flags="SHORTHAND_ALIAS_PROPERTY"
                     derive_serialize="True"
                     spec="Non-standard: https://developer.mozilla.org/en-US/docs/Web/CSS/transform">
-    use properties::longhands::transform;
-
     pub fn parse_value<'i, 't>(context: &ParserContext, input: &mut Parser<'i, 't>)
                                -> Result<Longhands, ParseError<'i>> {
+        use values::specified::transform::Transform;
         Ok(expanded! {
-            transform: transform::parse_prefixed(context, input)?,
+            transform: Transform::parse_internal(context, input, true)?,
```

Comment:
While you're touching this, changing the `prefixed` argument to an enum like:

```rust
enum ParsePrefixed {
    Yes,
    No,
}
```

would be neat, I think

Reply:
I'm not very keen to do this actually... Probably I can just move the `parse_prefixed` to `Transform`...